sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreProvider.scala

```diff
@@ -394,7 +396,7 @@ private[sql] class RocksDBStateStoreProvider
     rocksDB // lazy initialization
 
     val dataEncoderCacheKey = StateRowEncoderCacheKey(
-      queryRunId = getRunId(hadoopConf),
+      queryRunId = StateStoreProvider.getRunId(hadoopConf),
```

Comment:
Why do we change this ?

Reply:
This method used to be in the RocksDBStateStoreProvider object, I just moved it to the StateStoreProvider object since it is used for both HDFS and RocksDB state store providers